static/app/components/events/interfaces/stacktraceContentV2.tsx

```diff
@@ -239,15 +196,14 @@ function StackTraceContent({
             frame,
             isExpanded: expandFirstFrame && lastFrameIndex === frameIndex,
             emptySourceNotation: lastFrameIndex === frameIndex && frameIndex === 0,
-            nextFrame,
             prevFrame,
             platform,
             timesRepeated: nRepeats,
             showingAbsoluteAddress: showingAbsoluteAddresses,
             onAddressToggle: handleToggleAddresses,
             image: findImageForAddress(frame.instructionAddr, frame.addrMode),
             maxLengthOfRelativeAddress: maxLengthOfAllRelativeAddresses,
-            registers: {}, // TODO: Fix registers
+            registers: {},
```

Comment:
there is no need to fix registers here. I clarified this logic with Markus yesterday and he said that this is correct, as the registers should only passed to the last frame